src/main/java/com/ericsson/ei/controller/model/ParseInstanceInfoEI.java

```diff
@@ -156,19 +156,27 @@ public void init() throws IOException {
         private String enabled;
 
         @Getter
-        @Value("${ldap.url}")
+        @Value("${ldap.server.list}")
+        private String servers;
+
+        @Deprecated
```

Comment:
Supporting two ways/formats of properties is the same fault we got later in RemRem-publish, -> which ended up rewriting the code/properties handling to one format. So its better to have one format of properties, to avoid those issues in future. 
So choose one format of these properties, either these ldap properties or ldap.server.list property.

Reply:
But in this way we support backwards compatibility on the old configuraitons. As long as we test properly the old and new configurations and make sure both ways work I think it's better to keep both instead of introducting breaking changes in configuration. Right now this would have no impact on easy2use Eiffel bundle for example, since the old configuration still works

Reply:
It was exactly this that caused issues in RemRem,, where over time several properties was duplicated in different format and handled different and code was change that used these properties values. Later the different properties was read completely different, or more precised other format went into null point exception  or what it failed in. Specify deprecated should have an end date/remove date, if not, it just a word that will still be there and never be removed. To avoid month discussions and agree what format should be used and so on with customer, service teams, etc. about these issues, I prefer to support only one format already from beginning and  clearly inform in release not this has changed. Saves a lot of time,, since RemRem config issue has been discussed since June until now. So avoiding similar issues saves alot of time.

Reply:
There are no old properties anymore.